client/components/date-range-picker/helpers.js

```diff
@@ -80,21 +79,27 @@ export const getShortcuts = (maxDays, onClickHandler) => {
     options.sort((a, b) => a.daysAgo - b.daysAgo);
   }
 
-  options = options.map(option => ({
-    ...option,
-    onClick: () => onClickHandler(option),
-  }));
-
   return options;
 };
 
 export const getTimePanelLabel = showTimePanel =>
   showTimePanel ? 'Select date' : 'Select time';
 
-export const isDateValid = (date, minStartDate, maxEndDate) =>
-  date._isValid &&
-  date.isSameOrAfter(minStartDate) &&
-  date.isSameOrBefore(maxEndDate);
+export const isDateValid = (date, minStartDate, maxEndDate) => {
+  if (!date._isValid) {
+    return false;
+  }
+
+  if (date.isAfter(maxEndDate)) {
+    return false;
+  }
+
+  if (minStartDate && date.isBefore(minStartDate)) {
+    return false;
+  }
```

Comment:
These if statements can be merged together.

```
return date.isValid && !date.isAfter(maxEndDate) && !(minStartDate && date.isBefore(minStartDate));
```

Reply:
yea good point. I will make the change. Thanks!